CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ All notable, unreleased changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+- Product variant channel listing webhooks (stocks) - #SALEOR-2993 by @mstrumeck
```

Comment:
SALEOR-2993 is an internal task ID; in the changelog, we want to refer to PRs or commits that introduce described changes. Also, it may sounds too vague to the community; we want to describe as clearly as possible what has changed from their perspective. My suggestion:

```suggestion
- Add webhooks for stock changes: `PRODUCT_VARIANT_STOCK_CHANGED` and `PRODUCT_VARIANT_BACK_IN_STOCK`  - #7590 by @mstrumeck
```